Formula/awscli-ssm-plugin.rb

```diff
@@ -0,0 +1,20 @@
+class AwscliSsmPlugin < Formula
+  include Language::Python::Virtualenv
+
+  desc "Session Manager plugin for Amazon AWS command-line interface"
+  homepage "https://aws.amazon.com/systems-manager/"
+  url "https://s3.amazonaws.com/session-manager-downloads/plugin/1.1.54.0/mac/sessionmanager-bundle.zip"
```

Comment:
This looks macOS specific. Formula need to build from source and can't contained os specific binaries.

Reply:
Thanks @SMillerDev , let me check

Reply:
Ok, didn't see there was a Cask for it, sorry for the noise 😢